spec/requests/placements/schools/mentors/add_mentor_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "'Add mentor' journey", service: :placements, type: :request do
+  let(:current_user) { create(:placements_user, schools: [school]) }
+  let(:school) { create(:placements_school, :secondary, name: "Hogwarts") }
+  let(:school_id) { school.id }
+  let(:start_path) { new_add_mentor_placements_school_mentors_path(school_id:) }
+
+  before { sign_in_as current_user }
+
+  context "when starting a new wizard journey" do
+    before do
+      # Populate the wizard so it has some existing state
+      put step_path(:mentor), params: { "placements_add_mentor_wizard_mentor_step[first_name]" => "John" }
+    end
+
+    it "redirects to the first step of the wizard" do
+      get start_path
+      expect(response).to redirect_to step_path(:mentor)
+    end
+  end
```

Comment:
It's also worth testing that the wizard state gets reset here – otherwise there's no point in the `before do ... end` block which sets up pre-existing state.

For example, see:

https://github.com/DFE-Digital/itt-mentor-services/blob/16ffcd49135bb98dbb78134c3d5bcb23f9c392ee/spec/requests/placements/schools/placements/add_placement_spec.rb#L18-L22